cli/lib/errors.js

```diff
@@ -183,6 +183,25 @@ const invalidCacheDirectory = {
   `,
 }
 
+const insufficientPermissionOnSmokeTest = {
+  description:
+    'Cypress cannot use some directory due to file permissions',
+  solution: stripIndent`
+    If you are seeing this on some cache directory path
+
+      (e.g: ~/.cache or %userprofile%\AppData\Local\Cypress\Cache\<version>\Cypress)
+
+    then try to set CYPRESS_CACHE_FOLDER environment values, e.g.
+
+      CYPRESS_CACHE_FOLDER=<Your own directory> cypress install
+
+    and try again, 
+    Or, consider create a folder with your own permission first.
+    If you're working on Windows, See:
+    https://answers.microsoft.com/en-us/windows/forum/windows_10-files-winpc/give-permissions-to-files-and-folders-in-windows/78ee562c-a21f-4a32-8691-73aac1415373
+  `,
+}
```

Comment:
Fixing some grammar. We don't link to outside resources not under our control in our error messages. Since these error messages are permanent for all users on this version forever, we don't want to provide any dead links.

```suggestion
const insufficientPermissionOnSmokeTest = {
  description:
    'Cypress cannot access a directory due to file permissions',
  solution: stripIndent`
    If you're seeing this error when accessing a cache directory path

      (e.g: ~/.cache or %userprofile%\AppData\Local\Cypress\Cache\<version>\Cypress)

    try setting the \`CYPRESS_CACHE_FOLDER\` environment variable and install again.

      CYPRESS_CACHE_FOLDER=<Your own directory> cypress install

    Or consider creating a folder with the proper permissions first.`,
}
```